src/devices/bus/segaai/segaai_exp.h

```diff
@@ -0,0 +1,63 @@
+// license:BSD-3-Clause
+// copyright-holders:Wilbert Pol
+#ifndef MAME_BUS_SEGAAI_EXP_H
+#define MAME_BUS_SEGAAI_EXP_H
+
+#pragma once
+
+
+DECLARE_DEVICE_TYPE(SEGAAI_EXP_SLOT, segaai_exp_slot_device);
+
+
+class segaai_exp_interface;
+
+class segaai_exp_slot_device : public device_t,
+								public device_single_card_slot_interface<segaai_exp_interface>
+{
+public:
+	template <typename T>
+	segaai_exp_slot_device(const machine_config &mconfig, const char *tag, device_t *owner, T &&opts, const char *dflt, u32 clock)
+		: segaai_exp_slot_device(mconfig, tag, owner, clock)
```

Comment:
Please put the additional constructor arguments after the clock frequency so the first four arguments match the standard constructor as we do conventionally, i.e.
```c++
	template <typename T>
	segaai_exp_slot_device(const machine_config &mconfig, const char *tag, device_t *owner, u32 clock, T &&opts, const char *dflt, u32 clock)
```

Reply:
Ok, changed.